Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG BASE_IMAGE=signalwire/freeswitch-base:debian-10
```

Comment:
Since `libks` is a public repo we need to make sure the base images are publicly available as well, which is not the case today.